codebuild/bin/s2n_open_fds_test.py

```diff
@@ -0,0 +1,66 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Can you add a top level comment to the PR description showing what this will actually output when there is a leaked file descriptor?